Spring-server/build.gradle

```diff
@@ -0,0 +1,52 @@
+plugins {
+    id "org.springframework.boot" version "2.2.0.RELEASE"
+}
+
+apply plugin: 'java'
+apply plugin: 'org.springframework.boot'
+apply plugin: 'io.spring.dependency-management'
+
+sourceSets {
+    main {
+        resources {
+            srcDirs "src/main/resources", "config/"
+        }
+    }
+}
+
+task copyWebResources{
+    copy {
+        from '../assets'
+        into 'build/resources/main/public'
+    }
+}
+
+build.dependsOn copyWebResources
+
+bootJar {
+    baseName = 'post-client'
+    version = '0.0.1'
+    classifier = 'boot'
+}
+
+repositories {
+    mavenCentral()
+}
+
+sourceCompatibility = 11
+targetCompatibility = 11
+
+configurations {
+    compile.exclude module: "spring-boot-starter-tomcat"
+}
+
+dependencies {
+    compile "org.springframework.boot:spring-boot-starter-web"
+    compile 'org.springframework.boot:spring-boot-starter-undertow'
+    compile 'javax.mail:javax.mail-api:1.6.2'
+    compile "com.sun.mail:javax.mail:1.6.2"
```

Comment:
Do these two lines of code provide different classes? Can we delete one of them, or they are both needed?